velox/dwio/common/tests/utils/CMakeLists.txt

```diff
@@ -12,20 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(velox_dwio_common_test_utils BatchMaker.cpp DataFiles.cpp
-                                         DataSetBuilder.cpp FilterGenerator.cpp
-                                         UnitLoaderTestTools.cpp)
+add_library(
+  velox_dwio_common_test_utils
+  BatchMaker.cpp DataFiles.cpp DataSetBuilder.cpp FilterGenerator.cpp
+  UnitLoaderTestTools.cpp E2EFilterTestBase.cpp)
 
 target_link_libraries(
   velox_dwio_common_test_utils
   Folly::folly
+  fmt::fmt
+  glog::glog
+  gflags::gflags
+  gtest
   velox_dwio_common
   velox_dwio_common_exception
   velox_exception
+  velox_functions_prestosql
```

Comment:
why does this need presto functions?

Reply:
It provide utility to test `MetadataFilter`, which operates on complex logical expressions which involves comparison functions (e.g. `lt`, `eq`, `gt`) and logical negate `not`, these are all presto functions